runtime/Java/src/org/antlr/v4/runtime/VocabularyImpl.java

```diff
@@ -156,7 +156,7 @@ public int getMaxTokenType() {
 
 	@Override
 	public String getLiteralName(int tokenType) {
-		if (tokenType >= 0 && tokenType < literalNames.length) {
+		if (tokenType >= Token.MIN_USER_TOKEN_TYPE && tokenType < literalNames.length) {
```

Comment:
Token.MIN_USER_TOKEN_TYPE is more clear and is actually correct unlike previous 0